agent/grpc-external/services/dataplane/get_envoy_bootstrap_params.go

```diff
@@ -73,7 +74,24 @@ func (s *Server) GetEnvoyBootstrapParams(ctx context.Context, req *pbdataplane.G
 		NodeId:      string(svc.ID),
 	}
 
-	bootstrapConfig, err := structpb.NewStruct(svc.ServiceProxy.Config)
+	// This is awkward because it's designed for different requests, but
+	// this fakes the ServiceSpecificRequest so that we can reuse code.
+	_, ns, err := s.MergeNodeServiceWithCentralConfig(
+		nil,
+		store.(*state.Store),
```

Comment:
Would it be possible to have the method accept an interface scoped to the queries it needs to make? I'm guessing not because of the import-cycle problem?